### PR TITLE
Batch inference and test docs

### PR DESCRIPTION
**Pull request by hhaAndroid**
Batch inference and test docs

### CODE REVIEW

File: docs/1_exist_data_model.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ for frame in video:
 
 A notebook demo can be found in [demo/inference_demo.ipynb](https://github.com/open-mmlab/mmdetection/blob/master/demo/inference_demo.ipynb).
 
+Note: `inference_detector`only supported single image
```
Comment (ZwwWayne) [inline docs/1_exist_data_model.md:47]: Note: `inference_detector`only supported single-image inference for now.

File: docs/1_exist_data_model.md [hunk 2]
```diff
@@ -224,6 +226,8 @@ Optional arguments:
 - `--show-dir`: If specified, detection results will be plotted on the images and saved to the specified directory. It is only applicable to single GPU testing and used for debugging and visualization. You do NOT need a GUI available in your environment for using this option.
 - `--show-score-thr`: If specified, detections with scores below this threshold will be removed.
```
Comment (ZwwWayne) [inline docs/1_exist_data_model.md:227]: Also add explaination about `--cfg-options` and `--eval-options`.

File: docs/1_exist_data_model.md (hunk 1 above)
Comment (hellock) [reply docs/1_exist_data_model.md:47]: supports

File: docs/1_exist_data_model.md [hunk 3]
```diff
@@ -44,6 +44,8 @@ for frame in video:
 
 A notebook demo can be found in [demo/inference_demo.ipynb](https://github.com/open-mmlab/mmdetection/blob/master/demo/inference_demo.ipynb).
 
+Note: `inference_detector`only supports single-image inference for now.
```
Comment (hellock) [inline docs/1_exist_data_model.md:47]: Add a whitespace between "`" and "only".

File: docs/1_exist_data_model.md [hunk 4]
```diff
@@ -223,6 +225,10 @@ Optional arguments:
...
 - `--show-dir`: If specified, detection results will be plotted on the images and saved to the specified directory. It is only applicable to single GPU testing and used for debugging and visualization. You do NOT need a GUI available in your environment for using this option.
 - `--show-score-thr`: If specified, detections with scores below this threshold will be removed.
+- `--cfg-options`:  if specified, the key-value pair optional cfg will be merged into config file
+- `--eval-options`: if specified, the key-value pair optional eval cfg will be kwargs for dataset.evaluate() function, it's only for evaluation
+
+Test mode supports single and batch image, you can achieve by modifying `samples_per_gpu` in the config file. For example,  ``samples_per_gpu``in`configs/_base_/datasets/coco_detection.py`  is set to 1 to indicate a single image test,  and if set to greater than 1, it is the batch-test mode.
```
Comment (ZwwWayne) [inline docs/1_exist_data_model.md:231]: MMDetection supports inference with a single image or batched images in test mode. By default, we use single-image inference and you can use batch inference by modifying `samples_per_gpu` in the config of test data.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**